src/sentry/integrations/slack/message_builder/issues.py

```diff
@@ -243,7 +223,7 @@ def get_context(group: Group) -> str:
     return context_text.rstrip()
 
 
-def get_option_groups_block_kit(group: Group) -> Sequence[Mapping[str, Any]]:
+def get_option_groups(group: Group) -> Sequence[Mapping[str, Any]]:
```

Comment:
~generally return types should use concrete types (`list[...]`, `dict[...]`, etc.) -- this may also benefit from a `TypedDict` ?

Reply:
nice -- now with your TypedDict this should be `-> list[OptionGroup]`!